src/test/cli-integration/rbd/formatted-output.t

```diff
@@ -73,12 +73,10 @@ For now, use a more inclusive regex.
       "size": 1073741824, 
       "snapshot_count": 1
   }
-The version of xml_pp included in ubuntu precise always prints a 'warning'
-whenever it is run. grep -v to ignore it, but still work on other distros.
-  $ rbd info foo --format xml | xml_pp 2>&1 | grep -v '^new version at /usr/bin/xml_pp'
+  $ rbd info foo --format xml | xmlstarlet format -s 2 -o || true
```

Comment:
nit: may be use XMLSTARLET macro, which would fallback to `xml` if `xmlstarlet` is not available, as we have in other tests? Not sure if it is worth though.

Reply:
xmlstarlet is available on our testing distros for Octopus. I can't recall what distro had xml instead of xmlstarlet, do you?

Reply:
I think FreeBSD :-) but I don't think we run this test there (because I don't expect xml_pp to be there too).

Sure this suggestion can be ignored.